client/src/components/Filter/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './style.css'
+
+const Filter = ({ filterList, updateClickedFilter }) => {
```

Comment:
This can be simplified:
```
const Filter = ({ filterList, updateClickedFilter }) => (
  <React.Fragment>
    {filterList.map((filterType) => <button onClick={() => updateClickedFilter(filterType)}>{filterType</button>
    })}
</React.Fragment>
)
```